rapid_pro_tools/rapid_pro_client.py

```diff
@@ -286,6 +291,31 @@ def get_broadcast_for_broadcast_id(self, broadcast_id):
             f"(expected exactly 1)"
         return matching_broadcasts[0]
 
+    def interrupt_urns(self, urns):
+        """
+        Interrupts the given URNs from the flows they are currently in, if any.
+
+        Makes 1 request for every 100 URNs.
```

Comment:
I misunderstood this comment on first read to say that it was going to buffer requests and so wouldn't do anything until it had received 100 URNs.

Reply:
Changed to "If the list of URNs contains more than 100 items, requests will be made in batches of 100 URNs at a time."